docs/components/sensor/_index.md

```diff
@@ -37,7 +37,7 @@ For configuration information, click on one of the supported sensor models:
 | Model | Description |
 | ----- | ----------- |
 | [`fake`](fake/) | A model used for testing, with no physical hardware. |
-| [`ultrasonic`](ultrasonic/) | [HC-S204 ultrasonic distance sensor](https://www.sparkfun.com/products/15569) |
+| [`ultrasonic`](ultrasonic/) | An ultrasonic distance sensor like the [HC-S204](https://www.sparkfun.com/products/15569) |
```

Comment:
@bhaney Is this accurate? I realized that it may have been misleading to suggest one model of sensor in this place because this is such a common robotic component, but unsure. Can revert if you think more accurate

Reply:
@randhid is it accurate to say "like the HC-S204" or should we just recommend HC-S204?

Reply:
Ah yes our famous camera. So this technically refers to anything that uses an echo pin as an interrupt and a trigger pin interrupt. You can document that, there are people that will  read between the lines and grab another echo/trigger ultrasonic to use this module.  Calling out the HC-S204 is the most helpful thing for most people.

Also sorry for the name - maybe one day we can change this to its proper chip name. 

This model won't work with a single ultrasonic sensor without this chip - like the TCT40-16R, but might with another model that has both a trigger and echo pin - like the JSN-SR04T.